templates/base.libsonnet

```diff
@@ -63,8 +63,67 @@ local tpus = import "tpus.libsonnet";
         },
         spec: config.accelerator.PodSpec {
           local pod = self,
+          local commonEnv = [
+            {
+              name: "POD_NAME",
+              valueFrom: {
+                fieldRef: {
+                  fieldPath: "metadata.name"
+                },
+              },
+            },
+            {
+              name: "POD_UID",
+              valueFrom: {
+                fieldRef: {
+                  fieldPath: "metadata.uid"
+                },
+              },
+            },
+            {
+              name: "POD_NAMESPACE",
+              valueFrom: {
+                fieldRef: {
+                  fieldPath: "metadata.namespace"
+                },
+              },
+            },
+            {
+              name: "JOB_NAME",
+              valueFrom: {
+                fieldRef: {
+                  fieldPath: "metadata.labels['job-name']",
+                },
+              },
+            },
+            {
+              name: "MODEL_DIR",
+              value: 
+                "gs://xl-ml-test-us-central1/k8s/%(modelName)s/%(mode)s/%(acceleratorName)s/$(JOB_NAME)" % config,
```

Comment:
`us-central1` is hardcoded here but we already have a oneshots cluster in europe. It seems to me this wouldn't hard-break anything really: I guess our europe clusters might have longer read/write time to GCS. Would it cause any problems you can think of?

Reply:
For now, it will write summaries and checkpoints more slowly, which shouldn't be a huge deal. All of our automated tests still run in us-central1 for the moment. Long term, we should move this bucket name to a cluster-level ConfigMap to enable tests in other regions.